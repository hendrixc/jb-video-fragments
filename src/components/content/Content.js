import React, { Component } from 'react';
import './content.css';
import VideoPlayer from '../videoPlayer/VideoPlayer';

class Content extends Component {
  render() {
    return (
      <div className="content">
        <VideoPlayer />
      </div>
    );
  }
}

export default Content;
