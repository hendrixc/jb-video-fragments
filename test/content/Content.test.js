import React from 'react';
import renderer from 'react-test-renderer';
import Content from '../../src/components/content/Content';

it('renders an home component into the Content as snapshot', () => {
  const TextInputComponent = renderer.create(<Content />).toJSON();
  expect(TextInputComponent).toMatchSnapshot();
});
