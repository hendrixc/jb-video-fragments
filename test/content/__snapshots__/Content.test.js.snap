// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an home component into the Content as snapshot 1`] = `
<div
  className="content"
>
  asdasd
</div>
`;
